### DISCUSSION TITLE
Mausfunktionen für Cruiser Androide / Mouse functions for Cruiser Android

### DISCUSSION THREAD

**Discussion by Basti32** [devemux86/cruiser#696] [Q&A] [answered]
Hallo,
besteht die Möglichkeit, die Funktionen einer Maus in der Androidversion zu nutzen?
Folegender Hintergrund:
Auf Reisen geht nicht immer alle glatt und mir steht nur ein Tab für die Routenanpassung oder neue Routen abstecken zur Verfügung.
Jedes Mal wünschte ich mir bei diesen Arbeiten die Funktionen von einer Maus am PC, wenn ich mir Cruiser Desktop meine Routen plane.
Dabei liegt das Hauptaugenmerk auf das Scrollrad und rechte Maustaste.Zurzeit ist das Scrollrad (am PC der Zoom) ohne Funktion und die rechte Maustaste (am PC das Kontextmenü) gleicht der Zurücktaste am Tab.

Gibt es da Möglichkeiten, oder habe ich diese Funktion nur noch nicht entdeckt? 

Für mich wäre das eine deutliche Erleichterung.

Viele Grüße
Basti

Transalator DeepL

Hello, 
is it possible to use the functions of a mouse in the Android version? 
Background: 
When travelling, not everything always goes smoothly and I only have one tab available for route adjustment or setting out new routes. Every time I do this, I wish I had the functions of a mouse on the PC when I plan my routes with Cruiser Desktop.
At the moment, the scroll wheel (zoom on the PC) has no function and the right mouse button (context menu on the PC) is like the back button on the tab. 
Are there any options, or have I just not discovered this function yet? 
That would make things a lot easier for me. 
Best regards 
Basti

**Comment by devemux86** [comment]
@Basti32 

> is it possible to use the functions of a mouse in the Android version?

This is possible if you use Android apps on the desktop in an Android emulator. You can then use the mouse for all actions.

---

> besteht die Möglichkeit, die Funktionen einer Maus in der Androidversion zu nutzen?

Dies ist möglich, wenn Sie Android-Apps auf dem Desktop in einem Android-Emulator verwenden. Dann können Sie für alle Aktionen die Maus verwenden.

**Reply by Basti32** [reply]
Hallo @devemux86,

> Dies ist möglich, wenn Sie Android-Apps auf dem Desktop in einem Android-Emulator verwenden. Dann können Sie für alle Aktionen die Maus verwenden.

Dies kann ich leider nicht bestätigen. Die rechte Maustaste ist inaktiv und das Scrollrad schiebt die Karte nur hoch und runter.

Viele Grüße
Basti

Transalator DeppL:

Hello @devemux86, 

> This is possible if you use Android apps on the desktop in an Android emulator. You can then use the mouse for all actions. 

Unfortunately, I cannot confirm this. The right mouse button is inactive and the scroll wheel only moves the map up and down. 
Best regards 
Basti

**Reply by devemux86** [reply]
@Basti32 

The mouse wheel and right mouse button work on desktop apps, including desktop Cruiser.
Android Cruiser should be used on mobile devices.

Mausrad und rechte Maustaste funktionieren auf Desktop-Anwendungen, also auch auf Desktop Cruiser.
Android Cruiser sollte auf mobilen Geräten verwendet werden.

**Comment by devemux86** [comment, marked as answer]
@Basti32 

For best results, plan your routes with **Android Cruiser**.

On the desktop, it is better to use route planning websites such as [BRouter website](https://brouter.de/brouter-web/).
On the desktop Cruiser, the mouse wheel zooms the map and the right mouse button opens menus.

---

Die besten Ergebnisse erzielen Sie, wenn Sie Ihre Routen mit **Android Cruiser** planen.

Auf dem Desktop ist es besser, Routenplanungs-Websites wie [BRouter Website](https://brouter.de/brouter-web/) zu verwenden.
Auf dem Desktop Cruiser zoomt das Mausrad die Karte und die rechte Maustaste öffnet die Menüs.

**Reply by Basti32** [reply]
Hallo @devemux86,
danke für den Tipp, werde es aber erst mal mit Desktop Cruiser versuchen. 
Bei der BRouter Webseite sehe ich zur Zeit kein Mehrwert.
Ist mein Gedankengang richtig, dass die Routenberechnung Kurvig....... vom BRouter Profil bestimmt wird?
Viele Grüße
Basti

Transaltor DeppL:
Hi @devemux86, 
thanks for the tip, but I'll try Desktop Cruiser first. 
I don't see any added value with the BRouter website at the moment. 

Is my train of thought correct that the route calculation Kurvig....... is determined by the BRouter profile? 
Best regards 
Basti

**Reply by devemux86** [reply]
@Basti32 

Curvy routes are only available on Android Cruiser.

Kurvenreiche Strecken gibt es nur auf Android Cruiser.

**Reply by Basti32** [reply]
Hallo @devemux86 
> Kurvenreiche Strecken gibt es nur auf Android Cruiser.
Aber dies ist ein BRouter Profil?

Transaltor DeppL:

Hi @devemux86 

> Curvy routes are only available on Android Cruiser, 
but this is a BRouter profile?

**Reply by devemux86** [reply]
> Aber dies ist ein BRouter Profil?

@Basti32 Das ist richtig.

**Reply by Basti32** [reply]
@devemux86,
ist es möglich, dieses Profil /Profile in Desktop Cruiser einzupflegen?

Transaltor DeppL:

Is it possible to add this profile/profiles to Desktop Cruiser?

**Reply by devemux86** [reply]
> ist es möglich, dieses Profil /Profile in Desktop Cruiser einzupflegen?

@Basti32 No.
